lib/hanami/application/routing/middleware/stack.rb

```diff
@@ -20,6 +20,8 @@ class Stack
           ROOT_PREFIX = "/"
           private_constant :ROOT_PREFIX
 
+          attr_reader :stack
```

Comment:
```suggestion
          # @since 2.0.0
          # @api private
          attr_reader :stack
```

I suppose we should make this consistent with the rest of this class :)